frontend/workflows/projectSelector/src/project-group.tsx

```diff
@@ -41,16 +50,19 @@ const StyledMenuItem = styled.div({
 
 const StyledProjectHeader = styled.div({
   display: "flex",
-  alignItems: "center",
+  maxWidth: "100%",
+  alignItems: "flex-start",
   justifyContent: "space-between",
-  height: "48px",
+  minHeight: "40px",
   padding: "0 12px",
 });
 
-const StyledHeaderColumn = styled.div({
+const StyledHeaderColumn = styled.div((props: { grow?: boolean }) => ({
   display: "flex",
+  minHeight: "38px",
   alignItems: "center",
-});
+  flexGrow: props.grow ? 1 : 0,
```

Comment:
a lil confused by how this works. `grow` is passed to the div but I don't see a case when this can be false

Reply:
it's only passed as a bool prop on one instance of `StyledHeaderColumn`, default will be false

Reply:
oh! i missed that the div is used in other places 👍